app/assets/javascripts/views/story_view.js

```diff
@@ -546,6 +546,7 @@ module.exports = FormView.extend({
 
       this.renderNotes();
 
+      this.renderReactComponents();
```

Comment:
What about if you call here as `renderStoryControls` and let this function only for rendering, er... Story Controls?
This way we would not suggest to future development here, that all react rendering should be in this function, causing the risk we have a big function unnecessarily.

cc: @talyssonoc